pinakes/main/catalog/views.py

```diff
@@ -545,7 +548,7 @@ def cancel(self, request, pk):
 
 @extend_schema_view(
     retrieve=extend_schema(
-        tags=("orders", "order_items"),
+        tags=["orders", "order_items"],
```

Comment:
Because `drf_spectacular` type annotation declares that only list is expected.

https://github.com/tfranzel/drf-spectacular/blob/47cb4344314007649b6fa91fb08219a38e0b2d54/drf_spectacular/utils.py

Reply:
P.S. normally such annotation could look like `Sequence[str]` or `Iterable[str]`, but by some reason `drf_spectacular` declares `List[str]` explicitly.

P.P.S. List and tuple types have different semantics. Typically list is a sequence of homogeneous items that have common interface and processed in the same way, while tuple is a heterogeneous anonymous structure with access by-index. In Python however the line between them is blurred and these types are frequently interchangeable (tuples are frequently used as immutable sequences).